LoopUI/Views/LoopNotificationsView.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  LoopNotificationsView.swift
+//  LoopUI
+//
+//  Created by Rick Pasetto on 6/5/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import LoopKitUI
+import SwiftUI
+
+public struct LoopNotificationsView: View, HorizontalSizeClassOverride {
+    @Environment(\.dismiss) var dismiss
+    
+    private let backButtonText: String
+    @ObservedObject private var viewModel: LoopNotificationsViewModel
+    
+    private let notificationAndCriticalAlertPermissionScreen =
+        NotificationsCriticalAlertPermissionsView(viewModel: NotificationsCriticalAlertPermissionsViewModel())
+    
+    public init(backButtonText: String = "", viewModel: LoopNotificationsViewModel) {
+        self.backButtonText = backButtonText
+        self.viewModel = viewModel
+    }
+
+    public var body: some View {
+        NavigationView {
+            VStack {
+                List {
+                    notificationAndCriticalAlertPermissionSection
+                    supportSection
+                }
+                .listStyle(GroupedListStyle())
+                .navigationBarTitle(Text(LocalizedString("Loop Notifications", comment: "Loop Notifications settings screen title")))
+                .navigationBarBackButtonHidden(false)
+                .navigationBarHidden(false)
+                .navigationBarItems(leading: dismissButton)
+                .environment(\.horizontalSizeClass, horizontalOverride)
+            }
+        }
+    }
+    
+    private var dismissButton: some View {
+        Button( action: { self.dismiss() }) {
+            Text(backButtonText)
+        }
+    }
+            
+    private var notificationAndCriticalAlertPermissionSection: some View {
+        Section(header: SectionHeader(label: LocalizedString("Tidepool Loop Notifications", comment: "Section title for Tidepool Loop notifications"))) {
+            NavigationLink(destination: notificationAndCriticalAlertPermissionScreen) {
+                Text(LocalizedString("Notification & Critical Alert Permissions", comment: "Notification & Critical Alert Permissions button text"))
+            }
+        }
```

Comment:
[nit] Just FYI, but it was the use of `NavigationLink` and the ripple effect of updating view state based on observed objects that caused a number of issues for the CGM settings. If the view state update triggers on regular occurring events, e.g. incoming CGM measurements, it is possible the view and view model will update unexpectedly.

Reply:
Hm.  Okay, good to know.  This all will need rework anyway (post-510k) so I am not too concerned.